packages/fast-components-styles-msft/src/button/index.ts

```diff
@@ -111,18 +121,49 @@ const styles: ComponentStyles<ButtonClassNameContract, DesignSystem> = {
         background: neutralFillRest,
         "&:hover:enabled": {
             background: neutralFillHover,
+            [highContrastSelector]: {
+                background: "Highlight !important",
+                borderColor: "ButtonText !important",
+                color: "HighlightText !important",
+            },
+            "& $button_beforeContent, & $button_afterContent": {
+                fill: accentForegroundCut,
+                [highContrastSelector]: {
+                    fill: "HighlightText",
+                },
+            },
         },
         "&:active:enabled": {
             background: neutralFillActive,
         },
         ...applyFocusVisible<DesignSystem>({
             borderColor: neutralFocus,
+            ...highContrastOutlineFocus,
         }),
+        "&:disabled": {
+            [highContrastSelector]: {
+                background: "ButtonFace",
+                borderColor: "GrayText",
+                color: "GrayText",
+            },
+        },
         "&::-moz-focus-inner": {
             border: "0",
         },
-        "@media (-ms-high-contrast:active)": {
-            fill: "ButtonHighlight",
+        [highContrastSelector]: {
+            background: "ButtonFace",
+            borderColor: "ButtonText",
+            color: "ButtonText",
+            fill: "ButtonText",
+            "-ms-high-contrast-adjust": "none",
+        },
+        "a&": {
+            "&$button__disabled": {
+                "&:hover": {
+                    ...highContrastDisabledBorder,
+                    "& $button_beforeContent, & $button_afterContent": {},

```

Comment:
if this is empty it seems like you can remove it

Reply:
removed